jwst/resample/resample_utils.py

```diff
@@ -171,21 +171,19 @@ def _reproject(x, y):
     return _reproject
 
 
-def build_driz_weight(model, wht_type=None, good_bits=None):
+def build_driz_weight(model, weight_type=None, good_bits=None):
     """ Create input weighting image
     """
-    if good_bits is not None and good_bits < 0:
-        good_bits = None
     dqmask = build_mask(model.dq, good_bits)
     exptime = model.meta.exposure.exposure_time
 
-    if wht_type.lower()[:3] == 'err':
+    if weight_type == 'err':
         err_model = np.nan_to_num(model.err)
         inwht = (exptime / err_model)**2 * dqmask
         log.debug("DEBUG weight mask: {} {}".format(type(inwht), np.sum(inwht)))
-    # elif wht_type == 'IVM':
+    # elif weight_type == 'ivm':
     #     _inwht = img.buildIVMmask(chip._chip,dqarr,pix_ratio)
-    elif wht_type.lower()[:3] == 'exp':
+    elif weight_type == 'exp':
```

Comment:
Isn't the value that gets passed in for exposure time weighting "exptime", as opposed to "exp"? Or is that reset somewhere in the param parsing?

Reply:
Ah, thanks for catching that!  That's a bug.

Reply:
![minion_what](https://user-images.githubusercontent.com/12860084/42820938-602d340c-89a5-11e8-929a-f61a1cdae5e5.png)